packages/manager/modules/netapp/src/dashboard/volumes/dashboard/acl/controller.js

```diff
@@ -0,0 +1,112 @@
+export default class NetAppVolumesDashboardAclController {
+  /* @ngInject */
+  constructor($translate, Alerter) {
+    this.$translate = $translate;
+    this.Alerter = Alerter;
+  }
+
+  $onInit() {
+    this.selectRow(this.acls.length);
+    this.currentAcl = {
+      accessType: null,
+      accessTo: null,
+      accessLevel: null,
+    };
+  }
+
+  selectRow(index) {
+    this.currentEditedRow = index;
+  }
+
+  cancelAcl() {
+    this.isAdding = false;
+    this.selectRow(this.acls.length);
+    this.acls.pop();
+  }
+
+  addAcl() {
+    this.isAdding = true;
+    this.currentAcl = {
+      accessType: null,
+      accessTo: null,
+      accessLevel: null,
+    };
+    this.acls.push({});
+    this.selectRow(this.acls.length - 1);
```

Comment:
Let's say I have 4 rows, So previous selected row would be row number 4. Here we add an element so we now have 5 rows but we still select row number 4 ?

Reply:
Seen together.